app/src/androidTest/java/com/android/voyageur/ui/trip/activities/ActivitiesScreenTest.kt

```diff
@@ -176,4 +176,65 @@ class ActivitiesScreenTest {
         .assertIsNotDisplayed()
     verify(mockTripsViewModel).removeActivityFromTrip(sampleTrip.activities[1])
   }
+
+  @Test
+  fun clickingFilterButton_displaysFilterDialog() {
+    `when`(mockTripsViewModel.getActivitiesForSelectedTrip()).thenReturn(sampleTrip.activities)
+    composeTestRule.setContent {
+      ActivitiesScreen(navigationActions, userViewModel, mockTripsViewModel)
+    }
+    // Click the filter button
+    composeTestRule.onNodeWithTag("filterButton").performClick()
+    composeTestRule.onNodeWithTag("filterActivityAlertDialog").assertIsDisplayed()
```

Comment:
here can you perform a click somewhere else and check if the dialog stopped being displayed?